data/assets/scene/solarsystem/sun/heliosphere/mas/bastille_day/density_volume.asset

```diff
@@ -0,0 +1,49 @@
+asset.info = {
+    Name = "Volume Rendering Bastille Day",
+    Version = "1.0",
+    Description = [[ Volumetric rendering for the bastille day CME event ]],
```

Comment:
The `[[ ]]` was just in the example to show that you can use multiline comments as well